surveys/2024-04/questions.md

```diff
@@ -0,0 +1,160 @@
+# Survey April 2024
+
+## Demographics
+
+- Where are you from? (countries list if possible - or open question)
+  - country
+
+- How long have you been working with Node.js? (numbers list from 0 to 10+)
+
+- In what kind of organizations are you working in?
+  - big tech
+  - company
+  - startup
+  - university
+  - individual
+  - other
```

Comment:
one follow up question we wanted to ask is the core sector of the company, like banking, healthcare, ... 
This would be great to add it this year